EventFilter/L1TRawToDigi/interface/BxBlock.h

```diff
@@ -0,0 +1,70 @@
+#ifndef EventFilter_L1TRawToDigi_BxBlock_h
```

Comment:
hi @rekovic - this class is a data format - please move it to a DataFormats package

Reply:
(as are other classes in this package, but lets at least not add new ones)